ansible/roles/stack-sunbird/templates/sunbird_nodebb.env

```diff
@@ -1,10 +1,15 @@
 # Initialization variable for mongo
 # Ref: https://community.nodebb.org/topic/4325/set-up-nodebb-via-environment-variables
-database=mongo
+database=redis
+# redis database number
+redis_database={{nodebb_redis_database|default('0')}}
+# declearing js heap memory size
+nodebb_heap_size={{nodebb_heap_size|default('2048')}}
 secret={{nodebb_hash_key|default('1d57ba64-86d4-43ff-bd10-f6e9e0782899')}}
```

Comment:
What's this variable for? and why are we using a default value for the hash key?

Reply:
This is nodebb secret key and you only added this